src/main/java/br/com/autonomiccs/apacheCloudStack/client/ApacheCloudStackClient.java

```diff
@@ -192,15 +193,23 @@ protected HttpContext createHttpContextWithAuthenticatedSessionUsingUserCredenti
     }
 
     /**
+     * It createsa {@link RequestConfig} object with configured {@link #connectionTimeout} value.
+     * This can be used to set on a HttpClient its connect timeout, connection request timeout and socket timeout.
+     * @return
+     */
+    protected RequestConfig createRequestConfig() {
+        return RequestConfig.custom()
```

Comment:
There is a typo in the Java doc, "It createsa", I believe it is "It creates a".
Also, I think you can remove the "@return" as it is not being used there.
Could you also create a test case for this method? In my opinion, there is the need for a test to check if the `connectionTimeout` is, in fact, the timeout value configured by this method.